components/channel_move_to_submenu/index.tsx

```diff
@@ -74,11 +73,10 @@ const CategoryMenuItems = (props: Props) => {
             },
         }));
         trackEvent('ui', 'ui_sidebar_channel_menu_createCategory');
-    };
-
-    let filteredCategories = categories.filter((category) => category.type !== CategoryTypes.DIRECT_MESSAGES);
+    }
 
-    if (props.location === 'sidebar') {
+    let filteredCategories: ChannelCategory[] = [];
```

Comment:
For this specific "move to" option because this is a sub menu it will only get rendered once the menu is opened.

Reply:
Right. That's better than if it was at the top level like I think the previous one was.

I suppose we could make it so that this is only calculated when the submenu is opened, but that would have a much smaller effect than what this PR already has.